### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by AtillaTahak**
-added HTML list item for mobile menu
-added CSS for design for mobile menu
-added js for click event for mobile menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -845,3 +844,43 @@ textarea::placeholder {
     margin-left: 24px;
   }
 }
+
+.mobile-menu-list {
+  display: none;
+  flex-direction: column;
+  align-items: center;
```
Comment (Mihndim2020) [inline style.css]: - [x] Please according to the design, the mobile menu options are left aligned, not centered as in your implementation, please kindly fix that, as it is a requirement for this project and a great skill for a front end developer to stick to the design.  
***The design***
atilaD

***Your implementation***
atilaI

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: The project is running
:heavy_check_mark: Linters are passing

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const menu = document.getElementById('mobile-menu');
+const menuList = document.querySelector('.mobile-menu-list');
+const introPage = document.querySelector('.intro-page');
+const times = document.querySelector('.fa-times');
+const listitem = document.querySelector('.mobile-menu-list');
+menu.addEventListener('click', () => {
+  introPage.style.display = 'none';
+  menuList.style.display = 'flex';
+});
+times.addEventListener('click', () => {
```
Comment (mikenath223) [inline js/index.js:10]: - [x] The mobile menu still looks the same. Please fix the issue raised by the previous reviewer.

File: style.css [hunk 3]
```diff
@@ -845,3 +844,43 @@ textarea::placeholder {
...
+
+.mobile-menu-list-li {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  z-index: 1;
+  color: #fff5e1;
+  margin-bottom: 40px;
+  border-bottom: 2px solid #6f6c6b;
```
Comment (Mihndim2020) [inline style.css:877]: - [OPTIONAL] The border bottom covers almost the entire length of the screen, please kindly fix that. See image below.

***The design***
atilaD

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @AtillaTahak, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**